go/consensus/tendermint/full/full.go

```diff
@@ -821,6 +821,21 @@ func (t *fullService) GetStatus(ctx context.Context) (*consensusAPI.Status, erro
 			// We may not be able to fetch the genesis block in case it has been pruned.
 		}
 
+		lastRetainedHeight, err := t.GetLastRetainedVersion(ctx)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get last retained height: %w", err)
+		}
+		// Some pruning configurations return 0 instead of a valid block height. Clamp those to the genesis height.
+		if lastRetainedHeight < t.genesis.Height {
+			lastRetainedHeight = t.genesis.Height
+		}
+		status.LastRetainedHeight = lastRetainedHeight
+		lastRetainedBlock, err := t.GetBlock(ctx, lastRetainedHeight)
+		if err != nil {
+			return nil, fmt.Errorf("failed to fetch last reatianed block: %w", err)
```

Comment:
This shouldn't fail the method as otherwise no status will be returned. This currently fails the `e2e/early-query` test since there are no blocks available yet (and the status endpoint should work even when there are no blocks). Make sure to just not set the hash as above (and possibly set the last retained height to zero in this case as there are no blocks).

```
failed to get consensus status: failed to fetch last reatianed block: tendermint: block query failed: height 1 must be less than or equal to the current blockchain height 0
```

Reply:
this is guarded by t.started. is that something different?

Reply:
oh probably this is from the time between when it starts and when the first block is confirmed

Reply:
Yeah that's right.

Reply:
alright, changing it to handling similar to the above